database.sql

```diff
@@ -69,7 +69,7 @@ create table group_members (
 
 create table group_transactions (
 	group_id int not null,
-	transaction_id int not null,
+	transaction_id int not null unique,
```

Comment:
if a transaction can only be linked to one group, why have an associative entity? group_id could just be a null-able property of transactions